src/documenti/esterni/AdR/contenuti/casi_uso.tex

```diff
@@ -72,20 +72,36 @@ \subsection{UC4 - Visualizzazione errore ripristino sessione}
   \end{enumerate}
 \end{itemize}
 
-\section{UC2 - Selezione tipo di grafico}
+
+\section{UC3 - Selezione dimensioni}
+ \begin{itemize}
+     \item \textbf{Descrizione:} Selezione dimensioni con cui verrà visualizzato il grafico;
+     \item \textbf{Attore primario:} Utente;
+     \item \textbf{Precondizioni:} Il sistema è stato inizializzato [UC1];
+     \item \textbf{Postcondizioni:} Le dimensioni vengono aggiornate nel sistema e inizia la selezione del grafico [UC4];
+     \item \textbf{Scenario principale:}
+     \begin{enumerate}
+         \item Vengono mostrate all'utente le dimensioni di default e altre dimensioni tra cui scegliere;
+         \item Ogni dimensione presenta un apposito riquadro da selezionare nel caso la si voglia utilizzare;
```

Comment:
Analisi dei requisiti dovrebbe essere scollegato da progettazione e architettura, secondo me dicendo che sarà presente un apposito riquadro da selezionare illudendo ad un checkbox è come se stessimo già pensando come vogliamo venga l'applicativo esteticamente ma questo non è il compito dell'analista. Secondo me in questo punto dovresti semplicemente dire che l'utente può selezionare senza entrare nel dettaglio dicendo cosa vede l'utente